.github/workflows/build.yml

```diff
@@ -527,6 +527,8 @@ jobs:
       - uses: SwiftyLab/setup-swift@latest
         with:
           swift-version: '6.1'
+          # To be removed when swiftylab fixes the CI
```

Comment:
how do we enforce this to happen?

Reply:
https://github.com/google/flatbuffers/issues/8686